utils/frame/try-runtime/cli/src/lib.rs

```diff
@@ -788,6 +795,13 @@ pub(crate) fn state_machine_call_with_proof<Block: BlockT, D: NativeExecutionDis
 	let proof = proving_backend
 		.extract_proof()
 		.expect("A recorder was set and thus, a storage proof can be extracted; qed");
+
+	if let Some(path) = maybe_export_proof {
+		let mut file = std::fs::File::create(path)?;
+		use std::io::Write as _;
+		proof.using_encoded(|encoded_proof| file.write_all(encoded_proof))?;
```

Comment:
Is a binary block the best way to go here?  
What kind of analysis can you do on it?  
I am wondering if some structured JSON would be easier to analyze.

Reply:
@ggwpez I created a JSON format similar to the raw storage format in chain spec.